src/components/Button/Button.scss

```diff
@@ -5,6 +5,7 @@
   border-radius: 5rem;
   display: inline-block;
   height: 60px;
+  margin-right: 3rem;
```

Comment:
I don't think all our Buttons should have set margins. Did you mean to add this for a specific Button?